generators/app/templates/templatedPackage.json

```diff
@@ -73,12 +79,35 @@
     "libPathNodeModulesSymlink": "<%= libPathNodeModulesSymlink %>",
     "modelsNodeModulesSymlink": "<%= modelsNodeModulesSymlink %>"
   },
-  "jshintConfig": {
-    "camelcase": true,
-    "curly": true,
-    "devel": true,
-    "indent": 2,
-    "node": true
+  "eslintConfig": {
+    "env": {
+      "browser": true,
+      "node": true
+    },
+    "rules": {
+      "eqeqeq": 2,
+      "curly": 2,
+      "semi": 2,
+      "strict": [2, "never"],
+      "quotes": [2, "single"],
+      "no-trailing-spaces": ["error", {"skipBlankLines": false}],
```

Comment:
You can get most of it but some of it still comes from templated data which will have the wrong formatting.